### PR TITLE
Feat/348 automated releasing

### PR DESCRIPTION
**Pull request by eoln**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -268,41 +267,212 @@ jobs:
...
+          command: |
+            echo 'export DOCKER_ORG=`cat package.json | jq -r .name | sed s/@// | sed s/$/-inbound/ | sed s/\\\\/.*//`' >> $BASH_ENV
+            echo 'export CIRCLE_PROJECT_REPONAME=`cat package.json | jq -r .name | sed s/@// | sed s/$/-inbound/ | sed s/\.*\\\\///`' >> $BASH_ENV
+      - run:
+          name: Build Docker inbound:local
           command: |
-            echo "Building Docker image: $CIRCLE_TAG"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:$CIRCLE_TAG .
+            echo "Building Docker image-inbound: local"
+            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME/inbound:local -f ./docker/Dockerfile.Inbound ./
```
Comment (lewisdaly) [inline .circleci/config.yml:283]: Can you have another slash like this? 

e.g. `mojaloop/thirdparty-scheme-adapter/inbound:local` doesn't look like a proper docker image name + tag

File: .circleci/config.yml [hunk 2]
```diff
@@ -268,41 +267,212 @@ jobs:
...
+      - run:
+          name: Setup Docker env variables
+          command: |
+            echo 'export DOCKER_ORG=`cat package.json | jq -r .name | sed s/@// | sed s/$/-outbound/ | sed s/\\\\/.*//`' >> $BASH_ENV
+            echo 'export CIRCLE_PROJECT_REPONAME=`cat package.json | jq -r .name | sed s/@// | sed s/$/-outbound/ | sed s/\.*\\\\///`' >> $BASH_ENV
+      - run:
+          name: Build Docker outbound:local
+          command: |
+            echo "Building Docker image-outbound: local"
+            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME/outbound:local -f ./docker/Dockerfile.Outbound ./
```
Comment (lewisdaly) [inline .circleci/config.yml:304]: Same as above

File: .circleci/config.yml (hunk 1 above)
Comment (eoln) [reply .circleci/config.yml:283]: @lewisdaly yes there is more - let me fix it

**Review by lewisdaly — APPROVED** [review]
Much cleaner!

**Pull request merged**